wiki_articles/asan.md

```diff
@@ -10,39 +10,44 @@ such as:
 
 ## General Advice
 
-Not all sanitizers can be combined, but when they can, use e.g.:
+Not all sanitizers can be combined, but when they can, use e.g.:  
```

Comment:
Generally no, two spaces are actually the official way to introduce a line break. This could also be more explicit with `<br>`, I guess we could support some tiny subset of HTML here.

The line break is intentional in this place, because it's aesthetically better with how Discord wraps the text otherwise.

Reply:
We need to find a better solution here than relying on invisible characters. Note that many people's editors also automatically trim trailing whitespace on file save.

If you don't want to be as verbose as `<br>`, ` \` also works.

Reply:
All done now. I've decided to use `<br>` because of the weirdness of ` \`. GitHub displays the `\` when there is no preceding space, but displays a trailing space for ` \`. 

By comparison, StackOverflow doesn't display those trailing spaces. I don't like this inconsistency, and I'm not sure what the correct output is. `<br>` is also more flexible because it can be used anywhere, not just line endings.